pkg/kogito/server.go

```diff
@@ -197,6 +199,24 @@ func devSandboxHandler() func(w http.ResponseWriter, r *http.Request) {
 		r.Host = r.URL.Host
 
 		proxy := httputil.NewSingleHostReverseProxy(targetUrl)
+
+		// tolerate self-signed certificates
+		proxy.Transport = &http.Transport{
+			Proxy: http.ProxyFromEnvironment,
+			DialContext: (&net.Dialer{
+				Timeout:   30 * time.Second,
+				KeepAlive: 30 * time.Second,
+			}).DialContext,
+			ForceAttemptHTTP2:     true,
+			MaxIdleConns:          10,
+			IdleConnTimeout:       60 * time.Second,
+			TLSHandshakeTimeout:   10 * time.Second,
+			ExpectContinueTimeout: 1 * time.Second,
+			TLSClientConfig: &tls.Config{
+				InsecureSkipVerify: true,
```

Comment:
This creates a security issue as described in the [docs](https://pkg.go.dev/crypto/tls@go1.17#Config). I think that the best solution is to ask the user if he wants to disable this check, WDYT?. For now, it's not a problem, so I can create a JIRA to add this functionality eventually.

Reply:
Yeah. I understand. I believe the right way is to have this parameterized. Maye adding an option to config the service from the System Tray. Without this setting, the service can't connect to Openshift Cluster that uses self-signed certificates.